tests/visual/edit-dialog-form.html

```diff
@@ -0,0 +1,225 @@
+<!doctype html>
+<html lang="en" class="k-typography k-no-animations">
+
+<head>
+    <title>Edit dialog form</title>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="../../packages/default/dist/all.css" />
+    <link rel="stylesheet" href="assets/styles.css" />
+    <script src="assets/scripts.js"></script>
+
+    <style>
+        #test-area {
+            grid-template-columns: repeat(2, 1fr);
+        }
+
+        #test-area>section {
+            padding: 20px;
+            display: grid;
+            grid-template-columns: 1fr;
+            gap: 10px;
+        }
+        .k-dialog {
+            position: relative;
+        }
+    </style>
+</head>
+
+<body>
+
+    <div id="test-area" class="grid">
+
+        <section>
+
+            <div class="k-widget k-window k-dialog ng-trigger ng-trigger-dialogSlideInAppear" role="dialog" style="min-width:700px;0:min-width;min-width:700px;">
```

Comment:
You can remove some of the unneeded HTML. Do we need all the classes for the test? We definitely do not need the `role` attribute.
This applies for the whole test, not only for this line.